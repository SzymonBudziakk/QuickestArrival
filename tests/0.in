20 20
.........GDANSK.....
........*...........
........#...........
........#...........
*##################.
#SZCZECIN.........#.
#.................#.
##................#.
.############*#####.
.#...WARSZAWA.......
.#..................
.#############......
.#...........#......
.#..WROCLAW.##......
.#..*.......*.......
.####.......#KIELCE.
......*##.#########.
.OPOLE..#.*.......#.
........#.KRAKOW..#.
........###########.
2
KRAKOW GDANSK 10
WARSZAWA WROCLAW 3
3
KRAKOW GDANSK 1
WARSZAWA WROCLAW 1
SZCZECIN KRAKOW 1
