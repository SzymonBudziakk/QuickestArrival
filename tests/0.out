10
3
29 KIELCE
